argus/recipes/cloud/windows.py

```diff
@@ -205,6 +195,49 @@ def wait_cbinit_finalization(self):
         LOG.debug("Wait for the CloudBase Initit service to stop ...")
         self._backend.remote_client.manager.wait_cbinit_service()
 
+    def prepare_cbinit_config(self, service_type):
+        """Prepare the cloudbase-init config."""
+        self._cbinit_conf = cbinit_config.CBInitConfig(
+            client=self._backend.remote_client)
+
+        self._cbinit_unattend_conf = cbinit_config.UnattendCBInitConfig(
+            client=self._backend.remote_client)
+
+        # NOTE(mmicu)
+        # Disable first_logon_behaviour for testing purposes.
```

Comment:
Because first_logon_behaviour will control how the password
 should work on next logon, we could have failing tests due to authentication failures.